package.json

```diff
@@ -125,7 +122,6 @@
     "execa": "^0.10.0",
     "expiry-js": "0.1.7",
     "extract-text-webpack-plugin": "3.0.1",
-    "fetch-mock": "^5.13.1",
```

Comment:
Appears fetch-mock is used here https://github.com/elastic/kibana/blob/master/src/ui/public/error_auto_create_index/error_auto_create_index.test.js#L29